tests_integration/test_app_integration.py

```diff
@@ -1,56 +1,66 @@
 import pytest
-from dotenv import find_dotenv, load_dotenv
 import requests
-import app
+import json
+
+from dotenv import find_dotenv, load_dotenv
+from app import create_app
+from entity.item import Item
 
 @pytest.fixture
 def client():
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    
     # Create the new app.
-    test_app = app.create_app('.env.test')
+    test_app = create_app()
     
     # Use the app to create a test_client that can be used in our tests.
     with test_app.test_client() as client:
         yield client
 
-class MockCardsResponse:
-    def json():
-        return [
-            {
-                "id": "5ef7a4f991ad48544d0f2a53",
-                "idList": "todo_list_id",
-                "name": "This is a to do item",
-                "desc": "Description for to do item",
-                "dateLastActivity": "2020-07-30T12:52:06.278Z"
-            },
-            {
-                "id": "5ef7a4f991ad48544d0f2a54",
-                "idList": "doing_list_id",
-                "name": "This item is currently in progress",
-                "desc": "Description for item in progress",
-                "dateLastActivity": "2020-07-30T12:52:06.278Z"
-            },
-            {
-                "id": "5ef7a4f991ad48544d0f2a55",
-                "idList": "done_list_id",
-                "name": "This item is done",
-                "desc": "Description for done item",
-                "dateLastActivity": "2020-07-30T12:52:06.278Z"
-            },
-        ]
+def stub_get_db_collection():
+    return
 
-@pytest.fixture()
-def mock_get_requests(monkeypatch):
-    def mock_get(*args, **kwargs):
-        return MockCardsResponse
+def stub_get_all_items(collection):
+    return [
+        Item(
+            'test-to-do-item-id',
+            'Test To Do item Title',
+            'To Do',
+            'Test To Do item Description',
+            '2020-07-30T12:52:06.278Z'
+        ),
+        Item(
+            'test-doing-item-id',
+            'Test Doing item Title',
+            'Doing',
+            'Test Doing item Description',
+            '2020-07-30T12:52:06.278Z'
+        ),
+        Item(
+            'test-done-item-id',
+            'Test Done item Title',
+            'Done',
+            'Test Done item Description',
+            '2020-07-30T12:52:06.278Z'
+        )
+    ]
 
-    monkeypatch.setattr(requests, "request", mock_get)
+def test_app(monkeypatch, client):
+    monkeypatch.setattr(
+        'app.get_db_collection',
+        stub_get_db_collection
+    )
+    monkeypatch.setattr(
+        'app.get_all_items',
+        stub_get_all_items
+    )
 
-
-def test_app(mock_get_requests, client):
     response = client.get('/')
+    
+    content = str(response.data)
+    print (content)
```

Comment:
Probably here for debugging, but now it's working we should delete print statements